lib/facter/mysqld_version.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
 Facter.add('mysqld_version') do
-  confine { Facter::Core::Execution.which('mysqld') || Facter::Core::Execution.which('/usr/libexec/mysqld') }
+  confine { Facter::Core::Execution.which('mysqld') || Facter::Core::Execution.which('/usr/libexec/mysqld') || Facter::Core::Execution.which('/usr/local/libexec/mysqld') }
```

Comment:
mhm this becomes a bit messy. Do you know if we can set `PATH` for the `which()` method? Or is it a bug that it doesn't already check in `/usr/libexec/`?

Reply:
I don't know if there is a better way like setting a PATH before. If someone find a proper solution : that's ok to me.